sys/linux/dev_kvm.txt

```diff
@@ -232,12 +235,32 @@ kvm_text_x86_64 {
 	size	len[text, intptr]
 }
 
+# Unlike on other architectures, ARM64 text is a sequence of commands, each starting with
+# the call number and the command length.
 kvm_text_arm64 {
 	typ	const[0, intptr]
-	text	ptr[in, text[arm64]]
-	size	len[text, intptr]
+	text	ptr[in, array[syzos_api_call, 1:32]]
+	size	bytesize[text, int64]
+}
+
+syzos_api_uexit {
```

Comment:
We discussed that we don't need this in the descriptions, right?
Other API calls must exit on their own if we want to test them.